### PR TITLE
Return informative error message if any, but not all, of (ra, dec, radius) are provided in source filtering

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

File: skyportal/handlers/api/source.py [hunk 1]
```diff
@@ -78,7 +78,10 @@ def get(self, source_id=None):
                     [g.id for g in self.current_user.groups]))))
             if sourceID:
                 q = q.filter(Source.id.contains(sourceID.strip()))
-            if ra and dec and radius:
+            if any([ra, dec, radius]):
+                if not all([ra, dec, radius]):
+                    return self.error("If any of 'ra', 'dec' or 'radius' are "
```
Comment (stefanv) [inline skyportal/handlers/api/source.py]:
```suggestion
                    return self.error("If any of 'ra', 'dec' or 'radius' is "
```

**Comment by stefanv** [comment]
Good to go, apart from typo.

File: skyportal/handlers/api/source.py (hunk 1 above)
Comment (acrellin) [reply skyportal/handlers/api/source.py]: It's any number of, not just any single one, so I think "are" is right here ;)

Comment (stefanv) [reply skyportal/handlers/api/source.py]: Hah, I think the grammar breaks down either way... so guess it doesn't matter :joy:

**Pull request merged**